components/agentSearchForm/index.jsx

```diff
@@ -22,34 +22,38 @@
 import React, { useState } from "react";
 import { useId } from "react-id-generator";
 import T from "prop-types";
-import { Button, Input } from "@inrupt/prism-react-components";
+import { Form, Button, Input } from "@inrupt/prism-react-components";
 
-export function handleClick({ setAgentId, onSubmit }) {
-  return (agentId) => {
-    setAgentId("");
+function AgentSearchForm({ children, onSubmit, buttonText }) {
+  const [agentId, setAgentId] = useState("");
+  const inputId = useId();
+
+  const handleSubmit = () => {
     onSubmit(agentId);
+    setAgentId("");
   };
-}
 
-export function handleChange(setAgentId) {
-  return ({ target: { value } }) => {
-    setAgentId(value);
+  const handleChange = (event) => {
+    setAgentId(event.target.value);
   };
-}
 
-function AgentSearchForm({ children, onSubmit, buttonText }) {
-  const [agentId, setAgentId] = useState("");
-  const onClick = handleClick({ setAgentId, onSubmit });
-  const onChange = handleChange(setAgentId);
-  const inputId = useId();
   return (
-    <>
-      <Input id={inputId} label="WebID" onChange={onChange} value={agentId} />
+    <Form onSubmit={handleSubmit}>
+      <Input
+        id={inputId}
+        label="WebID"
+        onChange={handleChange}
+        value={agentId}
+        required
```

Comment:
Arne noted in #100 that `required` gave the textbox a red border when you load the page- we may want to remove this field and check it in onSubmit instead.